docs/.vuepress/config.js

```diff
@@ -0,0 +1,146 @@
+const path = require('path');
+const highlight = require('./highlight');
+const helpers = require('./helpers');
+const examples = require('./containers/examples');
+const sourceCodeLink = require('./containers/sourceCodeLink');
+
+const buildMode = process.env.BUILD_MODE;
+const environmentHead = buildMode === 'production' ?
+  [
+    // Google Tag Manager, an extra element within the `ssr.html` file.
+    ['script', {}, `
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+        '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+      })(window,document,'script','dataLayer','GTM-55L5D3');
+    `],
+  ]
+  : [];
+
+module.exports = {
+  patterns: ['**/*.md', '!README.md', '!README-EDITING.md', '!README-DEPLOYMENT.md'], // to enable vue pages add: '**/*.vue'.
+  description: 'Handsontable',
+  base: '/docs/',
+  head: [
+    ['link', { rel: 'icon', href: 'https://handsontable.com/static/images/template/ModCommon/favicon-32x32.png' }],
+    ['meta', { name: 'viewport', content: 'width=device-width, initial-scale=1' }],
+    // Cookiebot - cookie consent popup
+    ['script', {
+      id: 'Cookiebot',
+      src: 'https://consent.cookiebot.com/uc.js',
+      'data-cbid': 'ef171f1d-a288-433f-b680-3cdbdebd5646'
```

Comment:
Shouldn't 'data-cbid' be read from the env variable?

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik

Reply:
Reported https://github.com/handsontable/handsontable/issues/8428.